docs/book/install-and-build.sh

```diff
@@ -56,9 +67,10 @@ ${cmd} /tmp/mdbook.${ext}
 chmod +x /tmp/mdbook
 
 echo "grabbing the latest released controller-gen"
-go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.5
+go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0
 
 # make sure we add the go bin directory to our path
+echo "adding the go bin directory to our path"
```

Comment:
Unrelated change.

Reply:
required to fix the CI preview after the change and allow the merge
but you are right. I will split into another PR.